activerecord/lib/active_record/connection_adapters/abstract/schema_statements.rb

```diff
@@ -1773,10 +1773,14 @@ def validate_index_length!(table_name, new_name, internal = false)
 
         def validate_table_length!(table_name)
           if table_name.length > table_name_length
-            raise ArgumentError, "Table name '#{table_name}' is too long; the limit is #{table_name_length} characters"
+            raise ArgumentError, table_name_too_long_error_message(table_name)
           end
         end
 
+        def table_name_too_long_error_message(table_name)
+          "Table name '#{table_name}' is too long; the limit is #{table_name_length} characters"
```

Comment:
We usually end errors with periods. I noticed this was like that before, but let's add one now.

```suggestion
          "Table name '#{table_name}' is too long; the limit is #{table_name_length} characters."
```

Reply:
Thanks @eileencodes I've added them.